From 40ec13ec1dceda3eca65e30007c7e8690eadfbd7 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Fri, 14 Dec 2018 08:04:51 +0100
Subject: [PATCH 1/1] tests: enable unit tests by default on QEMU ARM(64)

Enable unit tests on QEMU ARM platforms by default so that we can test
fdt overlays.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 configs/qemu_arm64_defconfig | 1 +
 configs/qemu_arm_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/qemu_arm64_defconfig b/configs/qemu_arm64_defconfig
index 0f13716c2e8..0d4f43a5425 100644
--- a/configs/qemu_arm64_defconfig
+++ b/configs/qemu_arm64_defconfig
@@ -36,3 +36,4 @@ CONFIG_FLASH_CFI_DRIVER=y
 CONFIG_CFI_FLASH=y
 CONFIG_SYS_FLASH_USE_BUFFER_WRITE=y
 CONFIG_SYS_FLASH_CFI=y
+CONFIG_UNIT_TEST=y
diff --git a/configs/qemu_arm_defconfig b/configs/qemu_arm_defconfig
index b75363e305a..3f49c55650d 100644
--- a/configs/qemu_arm_defconfig
+++ b/configs/qemu_arm_defconfig
@@ -36,3 +36,4 @@ CONFIG_FLASH_CFI_DRIVER=y
 CONFIG_CFI_FLASH=y
 CONFIG_SYS_FLASH_USE_BUFFER_WRITE=y
 CONFIG_SYS_FLASH_CFI=y
+CONFIG_UNIT_TEST=y
-- 
2.19.2

